### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

**Comment by dlongley** [comment]
We need to hold review of this PR on a decision (or not) to store delegated zcaps and revoked zcaps (for the purpose of house keeping) in an EDV as opposed to on a server.

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
```
Comment (davidlehn) [inline .eslintrc.js:1]:
```suggestion
module.exports = {
  root: true,
```

Comment (mattcollier) [reply .eslintrc.js:1]: fixed

**Comment by dlongley** [comment]
I think we decided we don't need this PR. If true, we can close it.

**Pull request closed without merging**